src/common/ConfirmationModal.tsx

```diff
@@ -53,19 +52,19 @@ const ConfirmationModal: React.FC<ConfirmationModalProps> = ({ action, handleCli
       className={`popupModal${darkModeStatus ? "-dark" : ""} ${darkModeStatus ? "dark" : "light"}-theme${theme[darkModeStatus ? "dark" : "light"]}`}
     >
 
-      <h5>{header}</h5>
-      <p>Note: {note}</p>
+      <h5>{t(headerKey)}</h5>
```

Comment:
Please use <p> tag here and use class "popupModal-title" with inline style margin = 0

Reply:
made the changes